third_party/move/move-compiler-v2/tests/variable-coalescing/inlining1.exp

```diff
@@ -5,22 +5,30 @@ public fun Test::test(): u64 {
      var $t0: u64
      var $t1: u64
      var $t2: u64
-     var $t3: u64
   0: $t2 := 10
   1: $t1 := infer($t2)
-  2: $t3 := infer($t1)
-  3: $t0 := 3
-  4: return $t0
+  2: $t0 := 3
+  3: return $t0
 }
 
+
+Diagnostics:
```

Comment:
Hmm, we don't want these warnings?

Is Zekun's recent PR eliminating such warnings not merged in?